sidebars/components.js

```diff
@@ -1,9 +1,9 @@
 const backToHome = {
   extra: {
-    classNames: ['back-to-home', 'icon-small'],
+    classNames: ['back-to-home', 'iconSmall'],
     icon: 'img/arrow-left.svg',
   },
-  href: '/docs/welcome-to-libra',
+  href: '/docs/welcome-to-libra-v2',
```

Comment:
Can we remove the "v2"?

Reply:
@jjwill not if we want to keep the old file names the same. In that case I would have to rename them v1 or something along those lines